hack/common.sh

```diff
@@ -40,11 +41,25 @@ OSDK_WMCO_management() {
   local COMMAND=$1
   local OSDK_PATH=$2
 
-  $OSDK_PATH $COMMAND packagemanifests \
-    --olm-namespace openshift-operator-lifecycle-manager \
-    --operator-namespace openshift-windows-machine-config-operator \
-    --operator-version 0.0.0 \
-    --timeout 5m
+  if [[ "$COMMAND" = "run" ]]; then
+    # VERSION needs to be set in order to properly deploy the operator with operator-sdk run packagemanifests.
+    # This should always line up with the version specified in bundle/windows-machine-config-operator.package.yaml.
+    # for example windows-machine-config-operator.v2.0.0 in the package file, means VERSION=2.0.0.
```

Comment:
I though the package.yaml was deprecated. Is that not the case?

Reply:
I believe it may be when trying to distribute operators using it. However, package.yaml is required for `operator-sdk run packagemanifests`, as far as I can tell this method of testing is still supported. I cannot find a depreciation warning.